source/renderer/app/components/voting/voting-info/CurrentFund.js

```diff
@@ -0,0 +1,72 @@
+// @flow
+import React from 'react';
+
```

Comment:
@lucas-barros please remove this blank line 🙏

Reply:
Updated, thanks!